napari/_qt/_tests/test_plugin_widgets.py

```diff
@@ -192,6 +192,7 @@ def test_inject_viewer_proxy(make_napari_viewer):
     assert isinstance(wdg.viewer, PublicOnlyProxy)
 
     # simulate access from outside napari
-    with patch('napari.utils.misc.ROOT_DIR', new='/some/other/package'):
-        with pytest.warns(FutureWarning):
-            wdg.fail()
+    with patch(
+        'napari.utils.misc.ROOT_DIR', new='/some/other/package'
+    ), pytest.warns(FutureWarning):
+        wdg.fail()
```

Comment:
this is black doing its thing? ugh... Is it possible to get this instead?

```py
    with (
		patch('napari.utils.misc.ROOT_DIR', new='/some/other/package'),
		pytest.warns(FutureWarning),
	):
        wdg.fail()
```

Reply:
It is black and 80 character limits. I will try to work around this.

Reply:
The code that you suggest is available from python 3.10. We need to drop 3.8 and 3.9 to use it...

Reply:
^^' I vote in favour!

---

Jokes apart, I find the nested `with` way more readable than this, so I think I would rather disable this rule for now.

Reply:
I have reverted part of `SIM117` changes (keep this good-looking) and disabled this rule.